AlicerceTests/Logging/NodeJS/AlicerceLogServer.js

```diff
@@ -0,0 +1,34 @@
+//
```

Comment:
Agree. I think that we should test any network code via mocked networking and not have to rely on any local servers.